h2o-py/tests/testdir_algos/automl/pyunit_automl_args.py

```diff
@@ -8,7 +8,7 @@
 """
 This test is used to check arguments passed into H2OAutoML along with different ways of using `.train()`
 """
-def prostate_automl_args():
+def prostate_automl_args(): #TODO Why we call it `prostate`? It seems that main goal here is to check args no matter what dataset is being used.
```

Comment:
Many test names are not consistent.  The test names can be changed if it makes sense.

Reply:
I would change in another PR.

Reply:
So how is it being done... I can go and create Jira for that?

Reply:
@deil87 yes, you can create a JIRA for that. But its not high priority imho.

Reply:
@navdeep-G I do understand that. I'm just trying to figure out the rules :)

Reply:
Well, there is nothing stopping you from changing a test name. I am just saying it is not a high priority at this point.